### PR TITLE
feat: transient forcing checker

### PR DESCRIPTION
**Pull request by pgierz**
Adds a checker on the top-level CLI for the transient forcing table. Use as:

```console
$ esm_tools check-transient-forcing [table_path] [year]
```

If not provided, user is prompted. Uses the same machinery as the echam namelist.

### CODE REVIEW

File: src/esm_runscripts/namelists.py [hunk 1]
```diff
@@ -416,6 +416,9 @@ def echam_transient_forcing(config):
                         logger.error("echam:")
                         logger.error("    transient_forcing_table: /path/to/your/table")
                         sys.exit(1)
+                    # TODO(PG): This is a bit harsh, but until we get a better idea, dump the error:
+                    logger.error(e)
+                    breakpoint()
```
Comment (mandresm) [inline src/esm_runscripts/namelists.py]: Should this go as it is in `release` or do you intend to change it @pgierz?

File: src/esm_tools/cli.py [hunk 2]
```diff
@@ -58,5 +58,37 @@ def create_new_config(name, type):
     return 0
 
 
+# TODO(PG): This doesn't belong in the cli.py but whatever...
```
Comment (mandresm) [inline src/esm_tools/cli.py:61]: This is too echam-specific to be part of the ESM-Tools CLI, isn't it? We can discuss it on the monday meeting.

**Review by mandresm — COMMENTED** [review]
See comments below

File: src/esm_runscripts/namelists.py (hunk 1 above)
Comment (pgierz) [inline src/esm_runscripts/namelists.py]:
```suggestion
```

**Review by pgierz — COMMENTED** [review]
Removes breakpoint

**Review by mandresm — APPROVED** [review]

**Comment by pgierz** [comment]
#bump

**Pull request merged**